---
title: Discussion
format: 
    html:
        toc: true
        fontmain: "Georgia"
---


# Methods

## Alignment steps

Using bowtie2 (v.2.5.4), we created an index genome of **T. repens.** and aligned sequences to the reference genome. We then used the --un-conc-gz function to create a file of unaligned sequence. This sequence file was converted to forward and reverse, and is our exogenous, non t.repens DNA.

## Kraken2 Steps

We build a kraken2 database of bacteria, fungi, viral, and achaean DNA.

```{bash}
kraken2-build --download-taxonomy --db KrakenDB

kraken2-build --download-taxonomy bacteria --db KrakenDB
kraken2-build --download-taxonomy archaea --db KrakenDB
kraken2-build --download-taxonomy viral --db KrakenDB
kraken2-build --download-taxonomy fungi --db KrakenDB
kraken2-build --build --db KrakenDB
```


```{mermaid}
graph TD; 
    A[Kraken2] --> B[Braken2]
    A[Kraken2] --> C[MeAPHis-q]
    D[Centrifuge] --> C[MeAPHis-q]
    E[Clark-S] --> C[MeAPHis-q]

```

[Kraken2](https://github.com/DerrickWood/kraken2)
[Braken2](https://github.com/jenniferlu717/Bracken)
[Centrifuge](https://github.com/DaehwanKimLab/centrifuge)
[Clark (-S)](https://github.com/rouni001/CLARK)

# Development Forum

## Kraken2
- first is that we will want to start using the k2 wrapper instead. It will be supported longer. 
- next is that we'd want to use their 16S database alongside the bacteria, archaea, viral and fungal databases. We can test if using it alone or in combination works better, but that is available and we should use it. 